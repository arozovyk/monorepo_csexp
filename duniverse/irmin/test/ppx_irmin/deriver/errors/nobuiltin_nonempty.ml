type x = (unit[@nobuiltin ()]) [@@deriving irmin]
