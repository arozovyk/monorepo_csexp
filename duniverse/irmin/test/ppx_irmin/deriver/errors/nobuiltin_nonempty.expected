File "nobuiltin_nonempty.ml", line 1, characters 26-28:
Error: `nobuiltin` payload must be empty
